docs/documentation/release_notes/topics/26_0_0.adoc

```diff
@@ -202,6 +202,8 @@ The client libraries are these artifacts:
 
 It is possible that in the future, some more libraries will be included.
 
+The client libraries are supported with Java 8, so it is possible to use them with the client applications deployed on the older application servers.
```

Comment:
I think it would be good to mention that for all other components the required we support Java 17 and up, but perhaps this is not the place to do so. Perhaps we should handle this separately for the clients repo itself.

Reply:
Yes, true. But right now, we don't have any single place in the documentation with the mention of supported Java versions. I am not 100% sure if we want to add it and where to add? For product, I think that we will have the matrix with supported versions as usual.

Regarding the server, the Keycloak 25 release notes mention that OpenJDK 21 is supported and 17 is deprecated for the server, but apart from the release notes, I don't think that we have other place where the Java version is specified.

Reply:
Yeah, we could do a better job at this.